lightpath/ui/gui.py

```diff
@@ -1,6 +1,7 @@
 """
```

Comment:
No idea where it comes from, but lightpath prints this at the end of execution:
```
<lightpath.ui.gui.LightApp object at 0x7fb92a00a280>
```

Reply:
I think command-line entrypoints by default print the result of the function if not `None`.
We should probably remove the `return` from `return main(...)` in the entrypoint:
https://github.com/pcdshub/lightpath/blob/09be87d335952d39a597f2fe8bb1576bd8f2ece3/lightpath/main.py#L130

Reply:
I'm using pcds-5.4.2 + PYTHONPATH and running via `lightpath`.
If you don't get this print and the entrypoint is still wonky on that env then let's disregard this for now.

Reply:
This has puzzled me for a while, where exactly that printout came from. There is no obvious spot in the entrypoint mechanism. I believe this explains it:
```
$ python
Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:52)
[Clang 13.0.1 ] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.exit("puzzling printout")
puzzling printout
```

More tersely:
```
$ python -c "import sys; sys.exit('test')"; echo $?
test
1
```

And the `lightpath` script has:
```python
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(load_entry_point('lightpath', 'console_scripts', 'lightpath')())
```

Thus the return value from the entrypoint gets set as the argument to `sys.exit`. If it's not numeric, its string representation gets dumped to standard output and the process returns exit code `1`.

Removing the `return` from the entrypoint function should resolve this.

Reply:
that... is really bizarre and makes a lot of sense

Reply:
We should add this to the cookie cutter or something, I feel like several packages' entrypoints have needed Ken's intervention recently.  Nevertheless I'll toss an issue up for this 👍